wandb/sdk/wandb_login.py

```diff
@@ -327,6 +330,9 @@ def _login(
         )
         return False
 
+    if wlogin.use_identity_token():
+        return True
```

Comment:
Do we need any validation on the identity token within the file?

Reply:
We validate the audience, sub, issuer, expiration, and signature using the IDP's public key on the server side, before returning an access token. We are using the go package ory/fosite to help with this. This was mostly CVP's work and happy to point you to the go code if you're interested. Is there some validation you think should be added from a security standpoint on the client side?

Reply:
Nice, that seems good on the server side. I don't see any security concerns since this is just the client. Wondering what the user experience would be like if `_login()` returns True but the creds in `identity_token_file` are invalid

Reply:
Yeah - the login experience is kind of weird in the SDK. Right now "login" doesn't actually log you in at all. It just makes sure a credential exists (whether it's an API key or JWT). @kptkin told me this is something they plan on addressing in a future project.